url.bs

```diff
@@ -2247,6 +2247,18 @@ and then runs these steps:
      <li><p>If <a>c</a> is an <a>ASCII alphanumeric</a>, U+002B (+), U+002D (-), or U+002E (.),
      append <a>c</a>, <a lt="ASCII lowercase">lowercased</a>, to <var>buffer</var>.
 
+     <li>
+      <p>Otherwise, if <a>c</a> is U+003A (:) and <var>buffer</var>'s <a for=string>length</a> is 1 and 
+      <var>buffer</var> contains only an <a>ASCII alpha</a> and <a>remaining</a> starts with U+005C (\), then:
+
+      <ol>
+       <li><p>Set <var>url</var>'s <a for=url>scheme</a> to "<code>file</code>".
+       <li><p>Set <var>buffer</var> to the empty string.
+       <li><p>Replace every U+005C (\) code point in <a>remaining</a> with U+002F (/).
```

Comment:
Why is this needed?

Reply:
I think we still want to treat them as validation errors (note that those are not fatal). This entire feature is a legacy feature, it doesn't deserve to be valid or even partially valid.

Reply:
Thanks, I Updated to emit invalid-reverse-solidus validation errors for each backslash while still doing the replacement for legacy compatibility.